platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationFilesPoller.java

```diff
@@ -72,10 +72,22 @@ public ConfigurationFilesPoller(@NotNull Path configurationDirectoryPath,
         this.fileExtension = fileExtension;
     }
 
-    public void init() {
+    public void init() throws IOException {
         LOGGER.debug("Starting {}...",
                 this.getClass()
                         .getName());
+
+        try {
+            LOGGER.debug("Registering path [{}] with Watch Service.",
+                    configurationDirectoryPath.toString());
+            configurationDirectoryPath.register(watchService, ENTRY_CREATE);
+        } catch (IOException e) {
+            LOGGER.error("Unable to register path [{}] with Watch Service",
+                    configurationDirectoryPath.toString(),
+                    e);
+            throw e;
```

Comment:
Yeah I was thinking Eric just wanted it to be more clear when this fails from a debugging standpoint, but I wanted to make sure that was his intent 😄